packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Client-based rate limits
+excerpt: >-
+  Understand what client-based rate limits at Okta and learn how to design for efficient use of resources
+---
+
+# Client-based rate limits
+
+Client-based rate limiting for the `/authorize` endpoint provides granular isolation between requests made to the `/authorize` endpoint using a combination of the Client ID, user's IP address, and Okta device identifier. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+
+The `/authorize` [endpoint](/docs/reference/api/oidc/#authorize) is the starting point for OpenID Connect flows such as the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) or the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow). A request to this endpoint authenticates the user and either returns an authorization grant or tokens to the client application as part of the callback response.
```

Comment:
Maybe I'm wrong here, but isn't the authorization grant sent **from** the client and exchanged for the access token (plus refresh and id tokens if requested)?

Reply:
@IanHakes-okta You are right that `authorization grant` i.e authorization code is sent to the Authorization server (token endpoint) and is exchanged for Access, ID and Refresh tokens based on what was requested. But you would need to make the `/authorize` request in order to obtain the authorization code.

Reply:
so:

A request to this endpoint authenticates the user and either returns an authorization code or tokens to the client application as part of the callback response.